src/store/index.ts

```diff
@@ -237,14 +237,15 @@ const {
     },
 
     setNestedVariables(state, nestedVariables: NestedVariables) {
+      const newNestedVars = nestedVariables;
       // Remove duplicates from the nested variables
-      nestedVariables.bar = [...new Set(nestedVariables.bar)];
-      nestedVariables.glyph = [...new Set(nestedVariables.glyph)];
+      newNestedVars.bar = [...new Set(nestedVariables.bar)];
+      newNestedVars.glyph = [...new Set(nestedVariables.glyph)];
```

Comment:
I'd recommend defining this with an object literal (similarly to my suggestion above).

Reply:
Fixed in 84fc61c